Integrations/integration-GRR.yml

```diff
@@ -17,7 +17,7 @@ configuration:
   defaultvalue: "8000"
   type: 0
   required: false
-- display: Credentials
+- display: Username
```

Comment:
@anara123 would this cause backward compatibility issues ?

Reply:
Changing the display name will not break comptibility